### DISCUSSION TITLE
我想在java代码里访问paddleocr的模型，用于图片识别，这个要如何实现呢

### DISCUSSION THREAD

**Discussion by xuhongxin** [PaddlePaddle/PaddleOCR#11285] [General]
我想在java代码里访问paddleocr的模型，用于图片识别，这个要如何实现呢

**Comment by iammmmmmm** [comment]
在https://github.com/iammmmmmm/CodeExamples 仓库中有相关示例，具体位置在src/main/http/demo01。
不要忘记star~😘
